test/k8s/egress.go

```diff
@@ -408,6 +408,47 @@ var _ = SkipDescribeIf(func() bool {
 				})
 			})
 		})
+		Context("egress gw policy when the gateway is not found", func() {
```

Comment:
Too bad we are adding new ginkgo tests, but AFAIU egressgw tests are not yet ported to CI 3.0. I guess it's OK to add new test cases here then.